react/features/base/tracks/actions.js

```diff
@@ -795,6 +787,51 @@ export function destroyLocalDesktopTrackIfExists() {
     };
 }
 
+/**
+ * Create an action for when a screenshare track's muted state has been signaled to be changed.
+ *
+ * @param {(JitsiLocalTrack|JitsiRemoteTrack)} track - JitsiTrack instance.
+ * @returns {Function}
+ */
+export function screenshareTrackMutedChanged(track) {
+    return (dispatch, getState) => {
+        if (track.isMuted()) {
+            const sourceName = track.getSourceName();
+            const conference = getCurrentConference(getState());
+
+            dispatch(participantLeft(sourceName, conference));
+        } else {
+            dispatch(createFakeScreenShareParticipant(track));
+        }
+
+        return;
+    };
+}
+
+/**
+ * Create an action for when a screenshare track's video type state has been signaled to be changed.
+ *
+ * @param {(JitsiLocalTrack|JitsiRemoteTrack)} track - JitsiTrack instance.
+ * @param {VIDEO_TYPE|undefined} videoType - Video type.
+ * @returns {Function}
+ */
+export function screenshareVideoTypeChanged(track, videoType) {
+    return (dispatch, getState) => {
```

Comment:
This is the logic added to create a fake SS participant when a plan b endpoint SS